### PR TITLE
comment adder added!

### PR DESCRIPTION
**Pull request by Luke95G**

### CODE REVIEW

File: src/Components/AddComment.jsx [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+export const AddComments = ({ setComments, review_id}) => {
+    const [newComment, setNewComment] = useState("")
+
+    const handleSubmit = (event)=> {
+        event.preventDefault();
+        postComment(review_id, newComment)
+        .then((commentFromApi)=>{
+            setComments((currentComments)=>{
+                return [commentFromApi, ...currentComments]
+            })
```
Comment (itskatherine) [inline src/Components/AddComment.jsx:13]: Nice work updating state after a successful api post request

File: src/Components/AddComment.jsx [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+    return (<form className="comment" onSubmit={handleSubmit}>
+        <label htmlFor="newComment"></label>
+        <br></br>
+        <textarea className="commentBody" htmlFor="comment"
+        type="text"
+        value={newComment}
+        placeholder="Add a comment here..."
+        onChange={(event)=>setNewComment(event.target.value)} required/>
+        <br></br>
+
```
Comment (itskatherine) [inline src/Components/AddComment.jsx:27]: This form is a [controlled component](https://notes.northcoders.com/courses/js-front-end/controlled-components) - so will need another state to cover the input that is being typed (affected by onChange) and set to blank after being transferred over to another state (after submitting the comment) - check out the notes linked above for examples.
Currently the text stays in the textbox after submitted so I can spam the submit button to submit the same comment

File: src/Components/AddComment.jsx [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+<button className="Butt">Submit</button>
+    </form>)
+}
```
Comment (itskatherine) [inline src/Components/AddComment.jsx:30]: I can also post several times while waiting for a post request to happen, is there a way we can stop the user being able to submit a comment while the post request is in progress?

File: src/api.js [hunk 4]
```diff
@@ -31,3 +31,15 @@ export const patchReviewVotes = (review_id, inc_votes) => {
...
+  const postBody = {
+    username: "grumpy19",
+    body: body,
+  };
+  return gamesInstanceAPI
+    .post(`/reviews/${review_id}/comments`, postBody)
+    .then((response) => {
+      return response.data.newComment;
+    });
+};
```
Comment (itskatherine) [inline src/api.js:45]: Nice 👍

**Review by itskatherine — CHANGES_REQUESTED** [review]
Hi Luke, great stuff, just a few more things to consider before going forward - could you have a look at my comments and then merge this PR once you have? You can resubmit if you're not 100% sure 👍

File: src/Components/AddComment.jsx (hunk 2 above)
Comment (Luke95G) [reply src/Components/AddComment.jsx:27]: Thank you for the notes link!! Was trying to do this for ages last night lol, onto the next comment!! :tada:

**Pull request merged**